README.md

```diff
@@ -25,3 +25,11 @@ Commands:
   notify          makes a notify to pitaya server
   request         makes a request to pitaya server
 ```
+
+For connecting to a protobuffer server specify the documentation route with the -docs argument:
```

Comment:
It would be nice to have an example explaining how to use register push receivers for protobuf servers as well